test/test_sendgrid.py

```diff
@@ -24,7 +23,6 @@ def setUpClass(cls):
                 os.path.dirname(__file__)), '/..')
         cls.sg = sendgrid.SendGridAPIClient(host=host)
         cls.devnull = open(os.devnull, 'w')
-        prism_cmd = None
```

Comment:
It seems like both `import sys` and `prism_cmd` would be needed if that code below was ever uncommented though?

Reply:
Hello @42B  Thank you for reviewing the code.
Yes, I agree with you.
In my opinion, it seems good to remove the annotated code for Readability.
Ref:https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f

Reply:
@gy741 ,

Other than this one line, PR #653 seems to take care of this whole PR while also upgrading prism.

Reply:
Oh, Yes exactly,
Except for one thing, the prism_cmd variable was not removed.

Reply:
@gy741 

> "Other than this one line..."

Reply:
@42B Yes. :)
It would be nice to include it in PR #653.
 If you want, you will close me PR.